torch/_inductor/ir.py

```diff
@@ -2595,6 +2597,29 @@ def is_stride_order_storage_and_layout(
         return False
 
 
+def is_unaligned(node: IRNode) -> bool:
+    if isinstance(node, TensorBox):
```

Comment:
This and the next if statement could merged and the isinstance check could be combined?